pkg/webhooks/webhooks.go

```diff
@@ -181,3 +183,55 @@ func (wh WebhookHandler) ValidateDelete(hc *v1beta1.HyperConverged) error {
 
 	return nil
 }
+
+func (wh WebhookHandler) HandleMutatingNsDelete(ns *corev1.Namespace, dryRun bool) error {
+	wh.logger.Info("validating namespace deletion", "name", ns.Name)
+
+	operatorNsEnv, err := hcoutil.GetOperatorNamespaceFromEnv()
+	if err != nil {
+		wh.logger.Error(err, "failed to get operator namespace from the environment")
+		return err
+	}
```

Comment:
This is not going to changed. Maybe do it when creating the WebhookHandler object.

Reply:
done